601-spring-data-primitive-types/src/test/java/io/quarkus/qe/spring/data/CatResourceTest.java

```diff
@@ -0,0 +1,24 @@
+package io.quarkus.qe.spring.data;
+
+import io.quarkus.test.junit.QuarkusTest;
+import org.hamcrest.core.Is;
```

Comment:
There could probably be a static import here, just like in `BookResourceTest` :slightly_smiling_face:

Reply:
ok, will sync it :)